provider/blob/tests/test_versions.rs

```diff
@@ -63,3 +65,92 @@ fn test_v2() {
     let blob_provider = BlobDataProvider::try_new_from_blob(blob.into_boxed_slice()).unwrap();
     check_hello_world(blob_provider.as_deserializing());
 }
+
+// This tests that the V2Bigger format works by attempting to export something with 26^4 = 456976 data entries
+#[test]
+fn test_v2_bigger() {
+    // We do print progress since this is a slower test and it's useful to get some feedback.
+    println!("Exporting blob ....");
+    let mut blob: Vec<u8> = Vec::new();
+    let exporter = BlobExporter::new_v2_with_sink(Box::new(&mut blob));
+    DatagenDriver::new()
+        .with_keys([icu_provider::hello_world::HelloWorldV1Marker::KEY])
+        .with_locales_and_fallback([LocaleFamily::full()], Default::default())
+        .export(&ManyLocalesProvider, exporter)
+        .unwrap();
+
+    // Rather than check in a 10MB file, we just compute hashes
```

Comment:
Issue: The 10MB file is in this pull request.

Reply:
hm, thought I removed it